### PR TITLE
Update documentation for MercureBundle: add enable_profiler option

### PR DESCRIPTION
**Pull request by vincentchalamon**
Related feature: https://github.com/symfony/mercure-bundle/pull/13

panel

### CODE REVIEW

File: mercure.rst [hunk 1]
```diff
@@ -568,6 +568,17 @@ sent. Here is the PublisherStub implementation::
     App\Tests\Functional\Fixtures\PublisherStub:
         decorates: mercure.hub.default.publisher
 
+Debuging
+--------
+
+The MercureBundle now integrates a WebProfiler panel. To enable it, update your configuration as following::
```
Comment (OskarStark) [inline mercure.rst]:
```suggestion
The MercureBundle now integrates a WebProfiler panel. To enable it, update your configuration as following:

    .. code-block:: yaml
    
```

File: mercure.rst [hunk 2]
```diff
@@ -568,6 +568,17 @@ sent. Here is the PublisherStub implementation::
     App\Tests\Functional\Fixtures\PublisherStub:
         decorates: mercure.hub.default.publisher
 
+Debuging
```
Comment (OskarStark) [inline mercure.rst]:
```suggestion
Debugging
```

File: mercure.rst [hunk 3]
```diff
@@ -568,6 +568,17 @@ sent. Here is the PublisherStub implementation::
...
+--------
```
Comment (OskarStark) [inline mercure.rst]:
```suggestion
---------
```

**Review by OskarStark — COMMENTED** [review]
Thank you, just a few comments

**Review by OskarStark — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
This feature has been merged and released in MercureBundle 0.2. This PR can be merged in the 4.3 branch of the docs, because MercureBundle is a separate package already usable with Symfony 4.3.

File: mercure.rst [hunk 4]
```diff
@@ -568,6 +568,35 @@ sent. Here is the PublisherStub implementation::
...
+
+        # config/packages/mercure.yaml
+        mercure:
+            enable_profiler: '%kernel.debug%'
+
+    .. code-block:: xml
+
+        <!-- config/packages/mercure.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <mercure:config enable_profiler="%kernel.debug%"/>
```
Comment (wouterj) [inline mercure.rst]: Missing `<container>` + XSD/scheme references

File: mercure.rst [hunk 5]
```diff
@@ -568,6 +568,35 @@ sent. Here is the PublisherStub implementation::
     App\Tests\Functional\Fixtures\PublisherStub:
         decorates: mercure.hub.default.publisher
 
+Debugging
+---------
+
+The MercureBundle now integrates a WebProfiler panel. To enable it, update your configuration as following:
```
Comment (wouterj) [inline mercure.rst]: I think we can remove the "now", as the documentation is a guide and not so much of a changelog. We can add a `.. versionadded:: 0.2` "The WebProfiler panel was introduced in MercureBundle 0.2" directive at the start of this chapter though

**Review by wouterj — APPROVED** [review]
Thanks for the update. I left 2 comments, but both can be fixed while merging this PR.

File: mercure.rst (hunk 5 above)
Comment (vincentchalamon) [reply mercure.rst]: `.. versionAdded:: 0.2` doesn't seem to be accepted by the linter:
>mercure.rst ✘
>  571: You are not allowed to use version "0.2". Only major version "5" is allowed.
>   ->  .. versionadded:: 0.2
>  571: Please only provide ".. versionadded::" if the version is greater/equal "5.0"
>   ->  .. versionadded:: 0.2

Comment (wouterj) [reply mercure.rst]: Thanks for your very quick responses! 😍 You can ignore this lint message

@OskarStark what would you suggest doing here?

Comment (OskarStark) [reply mercure.rst]: I will add it to the ignore while merging and fix the `versionadded` directive 👍

**Comment by OskarStark** [comment]
Thank you Vincent.

**Pull request merged**